apps/cowswap-frontend/src/common/hooks/useBannerVisibility.ts

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
```

Comment:
Smth wrong with linting this file, we don't use semicolons

Reply:
This file has been removed in the follow up PR https://github.com/cowprotocol/cowswap/pull/3131